### DISCUSSION TITLE
Outpost Campaign Type

### DISCUSSION THREAD

**Discussion by Commander-Noob** [FakeFishGames/Barotrauma#8789] [Ideas and suggestions]
Like normal Campaigns, but instead of being on a submarine, you're on an outpost. You have to maintain an outpost, purchase things from passing submarines, sell stuff as well, give missions, take missions, defend against pirates, lockdown the outpost in the event of attacks from Abyssal Creatures and/or swarms of creatures. Fix and maintain submarines if the crew desires so. Upgrade your outpost in time for the Jovian Radiation, so that you may survive the wave of radiation it forces through the moon, waiting for the radiation to die down, so you can recover.

**Comment by James-E-A** [comment]
Even if this isn't something that looks maintainable or appealing to include as a long-term part of supported stock gameplay, it could nevertheless be a fun concept for an April Fool's event/temporary-update in 2023.

(And how much ability does the Modding system provide for a fan to do this themselves?)
